.github/workflows/test.yml

```diff
@@ -99,6 +99,7 @@ jobs:
           - '8.8.2'
           - '8.9.2'
           - '8.10.3'
+          - '8.11.0'
```

Comment:
Looks like something is not working when creating the custom metric SLO against this version of Kibana... Need to find the fix version, just testing out 8.11 for now.

Reply:
This looks to be unrelated... Could use some help with the errors in tests though 😅

Reply:
See 5ffaa40c401c9ef6d6dd7b1fbab59af6e3d8b765 🤦🏻